### DISCUSSION TITLE
[Feature] Allow Customize Favicon override for each feed

### DISCUSSION THREAD

**Discussion by johnsturgeon** [FreshRSS/FreshRSS#6502] [Ideas]
I get several RSS feeds from reddit and the favicon is always the same (resulting in the SAME favicon for every feed).  It would be great if I could override the favicon on a feed by feed basis.

I did see a similar question here #3875 but it was never turned into a formal feature request (not sure what the process for that is?).

**Comment by Alkarex** [comment]
https://github.com/FreshRSS/FreshRSS/issues/6503

**Comment by Alkarex** [comment]
Implemented in https://github.com/FreshRSS/FreshRSS/pull/7646
Tests welcome

**Comment by johnsturgeon** [comment]
Interesting that this was closed by the PR hasn't landed?

**Comment by Alkarex** [comment]
There were duplicated discussions and issues on the same topic. The issues are still open. Any feedback on the PR?

**Reply by johnsturgeon** [reply]
Gotcha!  No, PR looks great, I can't wait until it lands.  Thanks!

**Reply by Alkarex** [reply]
Does it work for you?

**Reply by johnsturgeon** [reply]
I'm on a vacation and have janky internet so I can't really pull the PR to test it.
